pro/src/pages/AdageIframe/app/components/OfferInfos/AdageOffer/utils/adageOfferStocks.ts

```diff
@@ -7,5 +7,7 @@ export function getBookableOfferStockPrice(
     style: 'currency',
     currency: 'EUR',
     minimumFractionDigits: 0,
-  }).format(offer.stock.price)} pour ${offer.stock.numberOfTickets} élèves`
+  }).format(
+    offer.stock.price / 100
```

Comment:
je me demande si on devrait pas dire que c'est le back qui nous renvoit le prix déjà divisé par 100 ?

Reply:
Oui c'est pas très pratique! C'est noté dans les chantiers de dette front/back EAC 👍